pandas/tests/series/methods/test_inplace_ops.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from pandas import Series
+import pandas._testing as tm
+
+
+@pytest.mark.parametrize(
+    "ser1, ser2, expected_add, expected_sub, expected_mul",
+    (
+        [
+            Series([1], dtype="Int64"),
```

Comment:
could you do the Series construction inside the test and just have the dtypes in the parameters?

Reply:
Done